google/ggrc-core
# GGRC-4527 Add OR operator to the export filter

# Issue description

OR option should be introduced in the filter by mapping.

# Steps to test the changes

1. Open Export page
2. Select filter by mapping multiple times
3. Check that OR option is available in dropdown
4. Select OR option for some cases
5. Click 'Start Export'
6. Check that generated csv contains correct data

# Solution description

Previously AND operator was hardcoded and the goal of this story was to add possibility to use operators dynamically. OR operator was added and now generated query depends on selected operators. Also all tests for exports components were in one place and they were moved to the different files for every component.

# Sanity checklist


# PR Review checklist
